Description: Disable evas test that fails outside of X
Author: Ross Vandegrift <ross@kallisti.us>
Forwarded: no
Last-Update: 2016-08-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/tests/evas/evas_test_image.c
+++ b/src/tests/evas/evas_test_image.c
@@ -393,6 +393,9 @@
    Eina_Strbuf *str;
    unsigned int i;
 
+   fprintf(stderr, "Skipping evas_object_image_all_loader_data\n");
+   return;
+
    obj = evas_object_image_add(e);
    ref = evas_object_image_add(e);
    str = eina_strbuf_new();
